src/python_pachyderm/client.py

```diff
@@ -343,6 +375,7 @@ def _parse_config(config):
         host = u.hostname
         port = u.port
         tls = u.scheme == "grpcs" or u.scheme == "https"
+        root_certs = bytes(root_certs, "utf-8") if root_certs is not None else None
```

Comment:
Interesting, is bytes a more robust way for passing the root_certs to gRPC? Just curious about why this change.

Reply:
root_certs is actually supposed to be bytes. It was incorrectly being returned from this helper function as a string.